go/libkb/upak_loader.go

```diff
@@ -294,6 +294,13 @@ func (u *CachedUPAKLoader) putUPAKToCache(ctx context.Context, obj *keybase1.Use
 	uid := obj.Current.Uid
 	u.G().VDL.CLogf(ctx, VLog0, "| Caching UPAK for %s %s", uid, stubMode)
 
+	// At this point, we've gone to the server, and we checked that the user is fresh, so if we previously had a stale
+	// bit set for this user, we'll turn it off now.
```

Comment:
We are sure this can't be called on any other code path other than after fetching from server?

Reply:
To the best of my knowledge.